images/cfssl/cloudbuild.yaml

```diff
@@ -2,7 +2,7 @@ timeout: 600s
 options:
   substitution_option: ALLOW_LOOSE
 steps:
-  - name: gcr.io/k8s-testimages/gcb-docker-gcloud:v20201130-750d12f
+  - name: gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20211008-60e346af
```

Comment:
/hold
I missed updating the tag in all of these other cloudbuild.yaml files, will amend the commit tomorrow